cmp/20f51f1d-ba6d-4319-961e-85645888661f/component.lp

```diff
@@ -0,0 +1,33 @@
+(librepcb_component 20f51f1d-ba6d-4319-961e-85645888661f
+ (name "Suppressor Diode Bidirectional Dual In Series")
```

Comment:
Why not use the name from the datasheet? "TVS Diode Arrays"?

Reply:
Because it is way too generic. See https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/53#issuecomment-562758022.